packages/frontend/src/pages/ParkingSpace/components/OfferRound.tsx

```diff
@@ -1,14 +1,139 @@
 import { Box, Chip } from '@mui/material'
 import type { GridColDef } from '@mui/x-data-grid'
-import { ApplicantStatus, LeaseStatus, OfferApplicant } from 'onecore-types'
+import {
+  ApplicantStatus,
+  LeaseStatus,
+  Offer,
+  OfferApplicant,
+} from 'onecore-types'
 
 import { DataGridTable } from '../../../components'
 import { OfferRoundActions } from './OfferRoundActions'
 
 export const OfferRound = (props: {
   numRound: number
   applicants: Array<OfferApplicant>
+  offer: Offer
 }) => {
+  const sharedProps = {
+    editable: false,
+    flex: 1,
+  }
+
+  const columns: GridColDef[] = [
```

Comment:
GridColDef moved inside the component to make the offer available as a prop.

Reply:
I moved it out for tidyness so will vote for `const getColumns = (params) => ...` :D